code/game/gamemodes/meteor/meteor.dm

```diff
@@ -69,6 +69,7 @@
 		universe.OnExit()
 
 	var/datum/universal_state/meteor_storm/meteor_storm = new /datum/universal_state/meteor_storm
+	universe = meteor_storm
```

Comment:
Nah, I had enough problems trying to retrofit the default proc and this just werks